### PR TITLE
 Update README

### PR DESCRIPTION
**Pull request by ms14981**
- Add IAM permissions needed by serverless.
- Add details of Gitlab integration.
- Add line explaining where to find the serverless logs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,38 @@ To deploy manually
...
-- Result should appear at the top of the page
+- Result should appear at the top of the page: ![Gitlab success message](./gitlab-success-message.png)
+- Add a `.clabot` file in the root of the project with a whitelist of the gitlab usernames that are cla-signed:
+```
+{
+  "contributors": ["<gitlab-username>"]
+}
+```
+- Get one of the whitelisted contributors to make a comment including the text "@gitlab-cla-bot check" on a merge request or create a merge request.
+
```
Comment (sehaswell) [inline README.md]: I think it shouldn't matter who comments on the MR - it's the committers who are checked against the whitelist.

Comment (sehaswell) [reply README.md]: Though I haven't checked this :)

Comment (ms14981) [reply README.md]: Would this text be more accurate?
```
- Create a merge request or make a comment including the text "@gitlab-cla-bot check" on a merge request and the bot should respond.
```

Comment (sehaswell) [reply README.md]: 👍

**Pull request merged**